app/javascript/packs/theme.js

```diff
@@ -57,6 +57,60 @@ const theme = createMuiTheme({
       secondary: "#DADADA",
     },
   },
+  props: {
+    MuiButton: {
+      disableElevation: true,
+      disableRipple: true,
+    },
+  },
+  buttons: {
```

Comment:
This is awesome! Love the idea of consolidating. Maybe we can make `StyledButton` similar to this: https://github.com/feedadoc/feedadoc/blob/master/app/javascript/packs/components/forms/StyledFormLabel.jsx#L5-L15

Or better yet, I think you can specify `MuiButton` in here and it will override global styles

Reply:
Good point! I'll look into the Material UI docs and see how to override global styles

Reply:
I moved the common styles to the overrides for `MuiButton` so it's applied to all! Left the others since the `text` styles need to be applied a surrounding div for the text (tried making the span for the button labels `display: block` but it didn't work) and the other button styles since we'll have more than just 2 styles (primary and secondary). I think having it this way will still help in consolidating and not have to repeat as much code!

Reply:
Nice work! Looks great!